src/orders/components/OrderTransactionActionDialog/OrderTransactionActionDialog.tsx

```diff
@@ -28,29 +29,34 @@ export const OrderTransactionActionDialog: React.FC<OrderTransactionActionDialog
   const actionType = actionIntl.toLowerCase();
 
   return (
-    <Dialog open={open} onClose={onClose}>
-      <DialogTitle>
-        <FormattedMessage
-          {...messages.title}
-          values={{
-            actionType,
-          }}
-        />
-      </DialogTitle>
-      <DialogContent>
-        <FormattedMessage {...messages.warningText} values={{ actionType }} />
-      </DialogContent>
-      <DialogActions>
-        <Button data-test-id="back" variant="secondary" color="text" onClick={onClose}>
-          <FormattedMessage {...buttonMessages.back} />
-        </Button>
-        <ConfirmButton
-          onClick={onSubmit}
-          labels={{ confirm: actionIntl }}
-          transitionState={confirmButtonState}
-          type="submit"
-        />
-      </DialogActions>
-    </Dialog>
+    <DashboardModal open={open} onChange={onClose}>
+      <DashboardModal.Content>
+        <DashboardModal.Title>
+          <FormattedMessage
+            {...messages.title}
+            values={{
+              actionType,
+            }}
+          />
+        </DashboardModal.Title>
+
+        <Text>
+          <FormattedMessage {...messages.warningText} values={{ actionType }} />
+        </Text>
+
+        <DashboardModal.Actions>
+          <Button data-test-id="back" variant="secondary" onClick={onClose}>
+            <FormattedMessage {...buttonMessages.back} />
+          </Button>
+
+          <ConfirmButton
+            onClick={onSubmit}
+            labels={{ confirm: actionIntl, error: actionIntl }}
```

Comment:
question: Are you sure about that, because there no translation for error in `mapActionToMessage[action]`

Reply:
Not sure I understand. We're not showing an error here anymore. `action` is always `CHARGE` and that translates to `Capture`.